doc/antora/modules/unlang/pages/xlat/predefined.adoc

```diff
@@ -780,7 +780,7 @@ Return named subcapture value from the last regular expression evaluated.
 
 Results of named capture groups are available using the `%{regex:<named capture
 group>}` expansion. They will also be accessible using the numbered expansions
-described <<_0_32,above>>.
+described link:_0_32[above].
```

Comment:
That's wrong and will break the in page linking.  The correct syntax for anchor links is ``<< >>``

Reply:
It's possible ``xref:predefined.adoc#_0_32[above]`` might work too.